hyrax-batch_ingest.gemspec

```diff
@@ -17,6 +17,10 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", "~> 5.1.6"
+  s.add_dependency "hyrax", "~> 2.2.0"
+  # This needs to be pinned to 11.5.2 because Hyrax won't install otherwise,
+  # because of some bizzare error eraised in cases where AF is *not* 11.5.2.
+  s.add_dependency "active-fedora", "11.5.2"
```

Comment:
It would be nice to relax the hyrax version restriction to capture 2.3.x versions.  The issue with AF 11.5.2 was resolved in Hyrax 2.3.1 and not backported to 2.2.x since there weren't any backwards compatible changes between 2.2.x and 2.3.x.

Reply:
i'll push an update presently

Reply:
Can we remove the `active-fedora` dependency now that the hyrax version is open to 2.3.1+?  I'm not sure how to allow the higher AF and lower AF but not mix AF 11.5.2 with hyrax < 2.3.1.  It is a complicated interplay between versions.

Reply:
blerg. Yes, sorry. forgot